### PR TITLE
add webp and zstd

### PR DESCRIPTION
**Pull request by bitner**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by gillins** [comment]
Is it possible to have Windows support (`bld.bat`)?

**Comment by bitner** [comment]
Unfortunately, I wouldn't know where to start with windows build. I can dig in and figure out if necessary to get this in.

**Comment by gillins** [comment]
Would be good to do since it looks like we build `libwebp` and `zstd` on Windows already. Have a look at https://github.com/OSGeo/gdal/blob/master/gdal/nmake.opt and search for webp and zstd to see what variables are required. Add them to `bld.bat`. Let me know if you have any trouble.

**Comment by gillins** [comment]
Please add a test for these drivers to `run_test.py` (https://github.com/conda-forge/gdal-feedstock/blob/master/recipe/run_test.py#L11)

File: recipe/set_bld_opts.bat [hunk 1]
```diff
@@ -86,5 +86,11 @@ set BLD_OPTS=%WIN64% ^
     POPPLER_MAJOR_VERSION=%POPPLER_MAJOR_VERSION% ^
     POPPLER_MINOR_VERSION=%POPPLER_MINOR_VERSION% ^
     POPPLER_LIBS="%LIBRARY_LIB%\poppler.lib"
+    WEBP_ENABLED=YES
+    WEBP_CFLAGS="-I%LIBRARY_INC%"
+    WEBP_LIBS="%LIBARY_LIB%\libwebp_a.lib"
```
Comment (gillins) [inline recipe/set_bld_opts.bat]: I think it is just `libwebp.lib` ?

BTW I suspect `libwebp` is not compiled for use in a DLL... Looking into.

Comment (gillins) [reply recipe/set_bld_opts.bat]: See https://github.com/conda-forge/libwebp-feedstock/pull/20

**Comment by gillins** [comment]
@conda-forge-admin, please rerender

**Comment by gillins** [comment]
Thanks @bitner I don't think we need anything else, just need to make the CI's pass.

@ocefpaf Any idea why we are getting this conflict on Windows (below)? I thought you rebuilt `libtiff`.

```
Package zstd conflicts for:
zstd=1.4.3
tiledb=1.7.0 -> zstd[version='>=1.4.3,<1.4.4.0a0|>=1.4.4,<1.4.5.0a0']
geotiff=1.5.1 -> libtiff[version='>=4.0.9,<5.0a0'] -> zstd[version='>=1.3.3,<1.3.4.0a0|>=1.3.7,<1.3.8.0a0|>=1.4.0,<1.4.1.0a0|>=1.4.3,<1.4.4.0a0|>=1.4.4,<1.4.5.0a0']
libtiff=4.0.10 -> zstd[version='>=1.3.3,<1.3.4.0a0|>=1.3.7,<1.3.8.0a0|>=1.4.0,<1.4.1.0a0']
poppler=0.67.0 -> libtiff[version='>=4.0.9,<5.0a0'] -> zstd[version='>=1.3.3,<1.3.4.0a0|>=1.3.7,<1.3.8.0a0|>=1.4.0,<1.4.1.0a0|>=1.4.3,<1.4.4.0a0|>=1.4.4,<1.4.5.0a0']
openjpeg=2.3 -> libtiff[version='>=4.0.9,<5.0a0'] -> zstd[version='>=1.3.3,<1.3.4.0a0|>=1.3.7,<1.3.8.0a0|>=1.4.0,<1.4.1.0a0|>=1.4.3,<1.4.4.0a0|>=1.4.4,<1.4.5.0a0']
```

**Comment by ocefpaf** [comment]
I guess we need to rebuild a few things. Working on this tomorrow. Probably `geotiff` b/c we have a new `libtiff`.

**Pull request merged**